tests/test_flows.sh

```diff
@@ -211,15 +211,31 @@ test_channel_selection() {
 }
 test_client_capability_query() { 
     status "test client capability"
+    sta_mac=00:00:00:11:00:22
 
     check_error=0
-    send_CAPI_1905 ${GATEWAY} $mac_agent1 0x8009 "tlv_type,0x90,tlv_length,\
-0x000C,tlv_value,{$mac_agent1_wlan0 0x000000110022}"
+    check send_CAPI_command ${GATEWAY} "DEV_SEND_1905,DestALid,$mac_agent1,MessageTypeValue,0x8009,tlv_type,0x90,tlv_length,\
+0x000C,tlv_value,{$mac_agent1_wlan0 0x000000110022}" $redirect
     sleep 1
     dbg "Confirming client capability query has been received on agent"
     # check that both radio agents received it,in the future we'll add a check to verify which radio the query was intended for.
-    check_log ${REPEATER1} agent_wlan0 "CLIENT_CAPABILITY_QUERY_MESSAGE"
-    check_log ${REPEATER1} agent_wlan2 "CLIENT_CAPABILITY_QUERY_MESSAGE"
+    check_log ${REPEATER1} agent "CLIENT_CAPABILITY_QUERY_MESSAGE"
+
+    dbg "Confirming that Client Capability Report message contains an Error Code TLV"
+    check_log ${REPEATER1} agent "Result Code: FAILURE"
```

Comment:
It would be better to check this on the controller instead of the agent. The commit that adds the handler on the controller could very easily parse out the result code and report it.

Also, to make sure the correct log message is checked, add the client MAC and BSSID to the output.